build/mod_md/patches/arc4random_buf.patch

```diff
@@ -0,0 +1,22 @@
+Patch origin: in-house
```

Comment:
I'd like to see this patch go, but after 3 hours of struggling with this I can't get past the compile error.

We have arc4random_buf in stdlib.h which is included in md_crypt.c, but for some reason the compiler is not seeing it. With the check removed from configure mod_crypto.c will fall back to a custom seed_RAND function.